extension/src/popup/styles/global.scss

```diff
@@ -45,6 +45,10 @@ a {
   height: 100%;
 }
 
+.simplebar-content {
```

Comment:
annoyingly this simple bar inner container doesn't grow to its parents height. I put it in global because I'm pretty sure we'd want this for any simplebar use

Reply:
🤔  could be ya . I ran through those views w/ this change and didn't see any noticeable differences, but good point will keep an eye on it